#! /bin/sh /usr/share/dpatch/dpatch-run
## 01path_li.dpatch by  <picca@synchrotron-soleil.fr>
## Public domain
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: gcc -> gcc-3.4 and remove all windows reference

@DPATCH@

--- ../path.li	2006-09-08 10:50:43.000000000 +0200
+++ ../../lisaac-0.84.orig/lisaac-0.84/path.li	2006-09-08 10:50:56.000000000 +0200
@@ -37,7 +37,6 @@
   
   + unix    := lib_os + "unix/";
   + dos     := lib_os + "dos/";
-  + windows := lib_os + "windows/";
     
   // Standard library:
   - lib + "time/";
@@ -67,30 +66,12 @@
   - lib + "gui/group_in/";
   - lib + "gui/group/";
   
-section WINDOWS,OS_WIN
-  
-  - windows + "file_system/";
-  - windows + "rsxnt/";
-  
-  * "gcc -Zwin32 " + input_file + " -o " + output_file + ".exe " + option_gcc;    
-  
-section DOS  
-  
-  - dos + "file_system/";
-  - dos + "int86/";
-  
-  * "gcc " + input_file + " -o " + output_file + ".exe " + option_gcc;
-  
-section UNIX, DOS, WINDOWS, OS_WIN, OS_LIN
+section UNIX, OS_LIN
   
   - unix   + "system/";
   - unix   + "file_system/";
-
-section UNIX, OS_LIN
-
-  - unix   + "x11/";
   
-  * "gcc " + input_file + " -o " + output_file + " -L/usr/X11R6/lib -lm -lX11 " + option_gcc;
+  * "gcc-3.4 " + input_file + " -o " + output_file + " -L/usr/X11R6/lib -lm " + option_gcc;
       
 
   
